supplementary_style_guide/glossary_terms_conventions/general_conventions/u.adoc

```diff
@@ -171,7 +171,7 @@
 [discrete]
 [[upstream-n]]
 ==== image:images/yes.png[yes] upstream (noun)
-*Description*: "Upstream" is data sent from a customer to a network service provider. Use the one-word form for the nominal form.
+*Description*: _Upstream_ is data sent from a customer to a network service provider. Use the one-word form for the nominal form.
```

Comment:
Duplicate of above

Reply:
The first is an adjective. The second is a noun.

Reply:
I looked multiple times to find the differences! Thank you :)